CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ release.
 
 ## [1.5.5]()
 ### Fixed
+- Fixed a bug where scale and offset were not being read or applied to GeoDataset objects.
```

Comment:
Should this be in an 'unreleased' section or are we cool with just putting it in 1.5.5?

Reply:
Yup, good catche! Fixed.